src/modules/Convolution/info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Convolution",
+    "description": "Image Convolution using a given 3x3 kernel matrix",
+    "inputs": {
+        "kernelValues": {
+            "type": "an array of 9 number values",
+            "desc": "values of the kernel which is used as mask for Convolution",
+        },
+        "pixels": {
```

Comment:
Shouldn't this be available to the module from sequencer itself?

Reply:
Yes, the pixels but values of the kernel will be taken from user. Sorry for that, will make the required changes!